packages/gafl-webapp-service/src/pages/contact/address/select/result-function.js

```diff
@@ -1,6 +1,14 @@
-import { CONTACT_SUMMARY_SEEN, CommonResults } from '../../../../constants.js'
+import { CONTACT_SUMMARY_SEEN, CommonResults, allowsPhysicalLicence } from '../../../../constants.js'
+import { isPhysical } from '../../../../processors/licence-type-display.js'
 
 export default async request => {
   const status = await request.cache().helpers.status.getCurrentPermission()
-  return status.fromSummary === CONTACT_SUMMARY_SEEN ? CommonResults.SUMMARY : CommonResults.OK
+  const permission = await request.cache().helpers.transaction.getCurrentPermission()
+  if (status.fromSummary === CONTACT_SUMMARY_SEEN) {
+    return CommonResults.SUMMARY
+  } else if (isPhysical(permission)) {
+    return allowsPhysicalLicence.YES
+  } else {
```

Comment:
I don't think you need this final else, if you omit it, it'll have the same result